### PR TITLE
rocksdb: support compaction_readahead_size for spinning disk

### PR DESCRIPTION
**Pull request by mrmiywj**
#1507 
Add compaction_readahead_size to the rocksdb config.

### CODE REVIEW

**Comment by mrmiywj** [comment]
PTAL @siddontang

**Comment by BusyJay** [comment]
Fix ci please.

**Comment by siddontang** [comment]
add this to the etc config template.

**Comment by siddontang** [comment]
please make format first.

**Comment by mrmiywj** [comment]
PTAL @siddontang @BusyJay @zhangjinpeng1987

File: etc/config-template.toml [hunk 1]
```diff
@@ -146,6 +146,11 @@ stats-dump-period-sec = 600
 # Default false.
 # disable-data-sync = false
 
+# Due to Rocksdb FAQ: https://github.com/facebook/rocksdb/wiki/RocksDB-FAQ,
+# If you want to use rocksdb on multi disks or spinning disks, you should set value at
+# least 2MB;
+compaction_readahead_size = 0
```
Comment (zhangjinpeng1987) [inline etc/config-template.toml]: s/compaction_readahead_size = 0/ # compaction_readahead_size = 0

**Comment by mrmiywj** [comment]
PTAL @zhangjinpeng1987

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by andelf** [comment]
@mrmiywj You've introduced first underscore-splitted option in TOML. 🥇

**Pull request merged**